libnavui-dropin/src/test/java/com/mapbox/navigation/dropin/NavigationViewListenerRegistryTest.kt

```diff
@@ -36,27 +38,36 @@ import org.junit.Test
 class NavigationViewListenerRegistryTest {
 
     @get:Rule
-    val coroutineRule = MainCoroutineRule()
-
-    private val testStore = TestStore()
-    private val maneuverBehaviorFlow =
-        MutableStateFlow<MapboxManeuverViewState>(MapboxManeuverViewState.COLLAPSED)
-    private val infoPanelBehaviorFlow = MutableStateFlow<Int?>(value = null)
-    private val mapClickBehaviorFlow = MutableSharedFlow<Point>(extraBufferCapacity = 1)
-    private val testListener = spyk(object : NavigationViewListener() {})
-    private val sut = NavigationViewListenerRegistry(
-        testStore,
-        mockk {
+    var coroutineRule = MainCoroutineRule()
+
+    private lateinit var sut: NavigationViewListenerRegistry
+    private lateinit var testStore: TestStore
+    private lateinit var maneuverBehaviorFlow: MutableStateFlow<MapboxManeuverViewState>
+    private lateinit var infoPanelBehavior: InfoPanelBehavior
+    private lateinit var mapClickBehavior: MapClickBehavior
+    private lateinit var testListener: NavigationViewListener
+    private lateinit var slideOffsetFlow: MutableStateFlow<Float>
```

Comment:
It is not necessary to move everything to `@Before`, because the test class is recreated before each test anyway.